.changeset/rude-radios-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/localized-money-input': minor
+---
+
+add precision badge to localized money input field
```

Comment:
```suggestion
Added a new property (**hasHighPrecisionBadge**) which can be used so the component shows a high precision badge when its value has a high precision decimal number.
```

Reply:
Thank you for the suggestion, updated here: https://github.com/commercetools/ui-kit/pull/2578/commits/e87ed73a1e7db4bb95602168669a7572b20d4bde